tasks.py

```diff
@@ -709,8 +709,21 @@ def hadolint(context):
 @task
 def pylint(context):
     """Run pylint code analysis."""
+    exit_code = 0
+
     command = 'pylint --init-hook "import nautobot; nautobot.setup()" --rcfile pyproject.toml nautobot_dev_example'
-    run_command(context, command)
+    if not run_command(context, command, warn=True):
+        exit_code = 1
+
+    # run the pylint_django migrations checkers on the migrations directory
+    migrations_pylint = (
+        'pylint --verbose --init-hook "import nautobot; nautobot.setup()" --rcfile pyproject.toml '
```

Comment:
Do we need the `--verbose`? I see you added that to the migrations_pylint command, but not to the other pylint command.

Reply:
Oh good catch! Core uses `--verbose` so I think we should add it to both commands.